README.md

```diff
@@ -180,6 +180,17 @@ A test server is running at <a href="coap://californium.eclipseprojects.io:5683/
 It is an instance of the [cf-plugtest-server](https://repo.eclipse.org/content/repositories/californium-releases/org/eclipse/californium/cf-plugtest-server/3.4.0/cf-plugtest-server-3.4.0.jar) from the demo-apps.
 The root resource responds with its current version.
 
+The server has a resource only accessible using OSCORE under "/oscore". It is configured with the following security material:
```

Comment:
Maybe add "(in preparation)", though it will take perhaps a couple of times, to deploy it to the sandbox after merging this PR.

Reply:
Done in commit ad70aba45b8a0006a67118fd0b8696d9211c0b1b